docs/configure-product/README.md

```diff
@@ -152,6 +152,17 @@ For the current configuration of your product, you can `curl` the API to retriev
 
 ### Configuring the `--product-resources`
 
+```json
+{
+  "job-name": {
+    "instances": "automatic",
```

Comment:
I tried to set the VMs instance count with the number as a string but Ops Manager returned a `422`. When this is set to an integer is works. Although `"automatic"` works, for example purposes, it would be better to specify this as some number since `"automatic"` is already the default.

I think it would be ideal to mix-in the assignment of `instance_type` and `persistent_disk` with the example manifest below. For instance, set `mysql`'s `persistent_disk` type to `204800`, and `consul_server`'s `instance_type` to `large.mem`.